Gemfile

```diff
@@ -83,7 +92,7 @@ group :development, :test do
 
   gem 'fictium'
 
-  gem 'byebug'
+  gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
```

Comment:
this option is only if you getting an uninitialized constant error

Reply:
use a normal way
```ruby
gem 'faker'
```

Reply:
ok!